kujira/assetlist.json

```diff
@@ -390,6 +390,44 @@
         }
       ]
     },
+    {
+      "description": "Quark's auto-compounding AQLA derivative",
+      "denom_units": [
+        {
+          "denom": "factory/kujira1d9fmsgvu3yvvppvqech2s3dhrqdm5ukngcf82ghcl70edfpwcrlsu6yr0u/urcpt",
+          "exponent": 0
+        },
+        {
+          "denom": "qcAQLA",
+          "exponent": 6
+        }
+      ],
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "base_denom": "factory/kujira1xe0awk5planmtsmjel5xtx2hzhqdw5p8z66yqd/uaqla",
```

Comment:
I don't see this. Can we register AQLA factory/kujira1xe0awk5planmtsmjel5xtx2hzhqdw5p8z66yqd/uaqla
as well?

Reply:
It's not my token, but I've added the info.